dspace/modules/additions/src/main/java/org/dspace/submit/step/ProjectStep.java

```diff
@@ -125,8 +116,11 @@ private int processProjectField(Context context, HttpServletRequest request, Ite
         String cv = request.getParameter(metadataField + "_confidence");
         if (StringUtils.isBlank(value) && noProjectAndFunderAttached(item)) {
             ProjectAuthorityValue project = new DSpace().getServiceManager().getServiceByName("defaultAuthorityCreator", DefaultAuthorityCreator.class).retrieveDefaultProject(context);
```

Comment:
Please refactor this into a class variable as done with the "projectService" variable.